seed/challenges/00-getting-started/getting-started.json

```diff
@@ -908,6 +1064,57 @@
+          "Ci vuole tempo per diventare bravi sviluppatori. Non ti aspetteresti di battere un campione di scacchi dopo aver giocato per 3 mesi. Non aspettarti di realizzare il prossimo Facebook dopo aver programmato per 3 mesi.",
+          ""
+        ],
+        [
+          "//i.imgur.com/EoTfOyC.jpg",
+          "Un'immagine di alcuni dei nostri camper che programmano insieme a Montreal.",
+          "Continua ad esercitarti a programmare tutti i giorni e discuti con altre persone che programmano, e potrai diventare uno sviluppatore pronto per il lavoro.",
+          ""
+        ],
+        [
+          "//i.imgur.com/EWWZBag.jpg",
+          "Un'immagine con il testo \" 1. Leggi l'errore 2. Cerca su Google 3. Chiedi aiuto. ",
+          "Ogni volta che ti trovi in ​​difficoltà o non sai cosa fare. Leggi-Cerca-Chiedi",
+          ""
+        ],
+        [
+          "//i.imgur.com/99BfAcK.jpg",
+          "Un'immagine che mostra la documentazione di jQuery",
+          "Innanzitutto, leggi la documentazione o il messaggio di errore. Una competenza chiave che i buoni programmatori hanno è la capacità di interpretare e seguire le istruzioni.",
+          ""
+        ],
+        [
+          "//i.imgur.com/GxvrsFb.gif",
+          "Una gif che mostra come fare una ricerca avanzata su Google. Prima di tutto, si inserisce la query \"jquery non viene eseguito quando la mia pagina si carica\". Poi si clicca sul pulsante strumenti di ricerca e si cambia l'opzione \"Qualsiasi data\" selezionando a \"Ultimo Anno\". Poi si clicca su un risultato, si legge l'articolo e troviamo la nostra risposta.",
+          "Se ciò non è stato d'aiuto, cerca su Google. Ci vuole un sacco di pratica per formulare delle buone query. Quando si esegue una ricerca su Google, di solito si preferisce includere il linguaggio o il framework che si sta utilizzando. Inoltre è meglio limitare i risultati ad un periodo recente . ",
+          ""
+        ],
+        [
+          "//i.imgur.com/LZYU7p2.gif",
+          "Una gif che mostra come seguire il link qui sotto per andare alla chat room di supporto e chiedere \"jquery non viene eseguito quando la mia pagina si carica\".",
+          "Se anche questo non ha aiutato, chiedi ai tuoi amici. In caso di problemi, si può chiedere ai tuoi amici campers nelle nostre chat rooms di supporto.",
+          "https://gitter.im/FreeCodeCamp/Help"
+        ],
+        [
+          "//i.imgur.com/ZRgXraT.gif",
+          "Una gif che mostra come scorrere la nostra mappa di problemi.",
+          "Ora sei pronto per iniziare a scrivere codice. Il tasto \"Mappa\" in alto a destra ti mostrerà la nostra mappa dei problemi. Ti consigliamo di completarli dall'alto verso il basso, ad un ritmo sostenibile. La nostra comunità open source sta migliorando costantemente i nostri problemi, quindi non stupirti se cambiano o se vengono spostati. Non si preoccupi di tornare indietro - continua ad andare avanti.",
```

Comment:
At the end it should be "Non preoccuparti di tornare indietro", the way it is right now is changing the person in the middle of the sentence, and does not agree with the person used in the previous sentences.